src/App.tsx

```diff
@@ -1,9 +1,26 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
+export function filterMovies(query: string) {
+  return (moviesFromServer.filter((movie) => {
+    const { title, description } = movie;
+
+    return (title.toLowerCase().includes(query.toLowerCase())
+    || description.toLowerCase().includes(query.toLowerCase()));
```

Comment:
one more thing, put this into separate function not to rewrite logic

Reply:
I really do not think I understand, let me try to apply this the way I understand it